wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/WrappersToGenerate.kt

```diff
@@ -319,6 +319,65 @@ val wrappersToGenerate = listOf(
             "push" to StringTyping,
         )
     ),
+    WrapperRequest(
+        ActionCoords("google-github-actions", "auth", "v0"),
+        mapOf(
+            "cleanup_credentials" to BooleanTyping,
+            "delegates" to ListOfTypings(","),
+            "token_format" to EnumTyping("TokenFormat", listOf("access_token", "id_token")),
+            "create_credentials_file" to BooleanTyping,
+            "access_token_scopes" to ListOfTypings(","),
+            "id_token_include_email" to BooleanTyping,
+        ),
+    ),
+    WrapperRequest(
+        ActionCoords("google-github-actions", "setup-gcloud", "v0"),
+        mapOf(
+            "export_default_credentials" to BooleanTyping,
+            "install_components" to ListOfTypings(
+                ",",
+                EnumTyping(
+                    "Component",
+                    // gcloud components list --format=json | jq -r '"\\"" + .[].id +"\","'
```

Comment:
There's a typo in the command (at least it fails on my machine) + let's keep it sorted:
```suggestion
                    // To regenerate, run: gcloud components list --format=json | jq -r '"\"" + .[].id + "\","' | sort
```